kie-bom/pom.xml

```diff
@@ -147,12 +147,12 @@
       <dependency>
         <groupId>org.kie.server</groupId>
         <artifactId>kie-server-services-optaplanner</artifactId>
-        <version>${version.org.kie}</version>
+        <version>7.73.0.Final</version>
```

Comment:
better to declare new version property just for optaplanner, please don't hardcode versions.